### ISSUE TITLE
Sort value changing after clicking header of discovered vms table

### ISSUE THREAD

**Issue by kedark3** [ManageIQ/miq_v2v_ui_plugin#343]
https://bugzilla.redhat.com/show_bug.cgi?id=1573963

**Comment by priley86** [comment]
i believe i have a fix for this - planning to update PF React dependency and we'll see what that does...

**Comment by JPrause** [comment]
@AllenBW can you add the blocker label for me.

**Comment by AllenBW** [comment]
@JPrause...

Never gonna give you up
Never gonna let you down
Never gonna run around and desert you
Never gonna make you cry
Never gonna say goodbye
Never gonna tell a lie and hurt you

unlike... say... @miq-bot 😏 🔪

**Comment by priley86** [comment]
yes - pf-react dependency update went in. also waiting on nightly feedback on this one. 

@kedark3 if you are still seeing this issue, can you please report back which browsers (user agent string would be great).

@AllenBW @JPrause not sure I understand what this is blocking, but hopefully it is fixed soon.

**Comment by bthurber** [comment]
Appears to be fixed in latest nightly.
